### DISCUSSION TITLE
Contribute to `libspdm`

### DISCUSSION THREAD

**Discussion by PrithviAPai** [DMTF/libspdm#2594] [General]
I tried pushing some changes to libspdm 
But I see it is being denied. 
Should repo owners need to add contributors' name to name list ? 
`
git push -u origin spdm-move-send-receive-to-common
remote: Permission to DMTF/libspdm.git denied to PrithviAPai.
fatal: unable to access '[https://github.com/DMTF/libspdm.git/':](https://github.com/DMTF/libspdm.git/%27:) The requested URL returned error: 403
`
I m able to clone, pull etc from the repo.

Thanks,
Prithvi A Pai

**Comment by jyao1** [comment]
Push is forbidden. Please submit PR, if you want to change anything.

**Reply by steven-bellock** [reply]
In particular you will need to [create a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo) of libspdm. For example : https://github.com/steven-bellock/libspdm/.
